psake-build-helpers.ps1

```diff
@@ -0,0 +1,44 @@
+function Set-Project-Properties($versionPrefix) {
+    write-host "$versionPrefix"
+
+    set-regenerated-file "$pwd/Directory.Build.props" @"
+<Project>
+    <PropertyGroup>
+        <VersionPrefix>$versionPrefix</VersionPrefix>
```

Comment:
Don't use `<VersionPrefix>` here, use `<Version>` to apply the entire version. It will extract the prefix and suffix automatically assuming a valid semantic version.